content/updates/2020-09/1-API-search.md

```diff
@@ -0,0 +1,191 @@
+---
+pim_announcement_img: ./img/api-filters.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New updates to ease API catalog structure synchronization
+::: meta-data type="New" features="API, Connectivity" available="early September" link-to-doc="https://api.akeneo.com/documentation/filter.html"
+:::
+
+We added new filters on the `search` parameter which enables you to add filters to catalog structure endpoints. Moreover, we added **attribute group labels** inside the attribute endpoints.  
+
+Those two enhancements will help reduce further the number of data to process, the size of the API call responses, and the number of API calls needed to synchronize your catalog structure. 
+
+## The new filters
+
+Here are the new filters:
+- **updated date**: allows you to retrieve only data updated since a specific date and hour (now also available for attribute groups), 
+- **code**: allows you to filter on specific codes (now also available for attribute groups and categories), 
+- **parent**: allows you to retrieve all sub-categories of a parent category (only available for categories).  
+
+Let's take an example: you have several category trees in your PIM, and you want to synchronise only the `eCommerce` category tree with your eCommerce backend.
+
+The following filter enables you to retrieve only the categories that are in the `eCommerce` category tree:
+
+```
+/api/rest/v1/categories?search={"parent":[{"operator":"=","value":"ecommerce"}]}
+```  
+
+::: tips
+You can combine filters to achieve powerful results!
+:::
+ 
+## Attribute group labels inside the attribute endpoints  
+Imagine you want to share your product data thanks to your online catalog.  
```

Comment:
I don't understand the example.. you're talking about a CMS or an eCom platform here?

Reply:
I was thinking about an online catalog like the Ikea catalogue : https://publications-fr-fr.ikea.com/ikea_catalogue/page/1

Reply:
Or a product catalogue that could be seen on tablets by in-store sales to inform customers on product characteristics.